templates/adding_a_new_model/cookiecutter-template-{{cookiecutter.modelname}}/modeling_tf_{{cookiecutter.lowercase_modelname}}.py

```diff
@@ -327,7 +336,6 @@ def call(self, hidden_states, input_tensor, training=False):
         return hidden_states
 
 
-# Copied from transformers.models.bert.modeling_tf_bert.TFBertLayer with Bert->{{cookiecutter.camelcase_modelname}}
```

Comment:
I unfortunately had to remove this, even though it's still the same as BERT; the reason being the way the variable is called in the model:

```py
self.{{cookiecutter.lowercase_modelname}}_output = TF{{cookiecutter.camelcase_modelname}}Output(config, name="output")
```
in BERT, therefore
```py
self.bert_output = TFBertOutput(config, name="output")
```
here. The `# Copied from ...` util doesn't seem to allow this. I can put the tag on the `call` method, but I wanted to check with you @sgugger if we can't allow this here since it's the model name that's creating an issue.

Reply:
Looks like it's just a regex to fix.